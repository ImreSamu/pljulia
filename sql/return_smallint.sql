CREATE FUNCTION julia_increment(x INTEGER)
RETURNS SMALLINT AS $$
    x + 1
$$ LANGUAGE pljulia;
SELECT julia_increment(1);
DROP FUNCTION julia_increment(x INTEGER);
