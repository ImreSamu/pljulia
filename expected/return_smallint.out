CREATE FUNCTION julia_increment(x INTEGER)
RETURNS SMALLINT AS $$
    x + 1
$$ LANGUAGE pljulia;
SELECT julia_increment(1);
 julia_increment 
-----------------
               2
(1 row)

DROP FUNCTION julia_increment(x INTEGER);
